module-2/README.md

```diff
@@ -245,10 +246,10 @@ After your service is created, ECS will provision a new task that's running the
 
 #### Test the Service
 
-Copy the DNS name you saved when creating the NLB and send a request to it using the preview browser in Cloud9 (or by simply any web browser, since this time our service is available on the Internet). Try sending a request to the mysfits resource:
+Copy the DNS name you saved when creating the NLB and send a request to it using the preview browser in Cloud9 (or by using any web browser or CLI tool, since this time our service is available on the Internet). Try sending a request to the mysfits resource:
```

Comment:
The instructions here are a bit contradictory now. The directions say use the preview browser and then the command shows curl.  I don't think the change from using the built-in preview browser or curl has added much here and isn't a needed change (since the preview browser will give more exposure to the features of Cloud9 rather than a generic curl command - regardless of which one will be more commonly used in practice :) ).